EasySettings/SettingsTab.cs

```diff
@@ -335,6 +335,34 @@ private static void RegisterKeyButton(AtlyssKeyButton element)
         manager.keyBindButtons = manager.keyBindButtons.AddToArray(keyButton);
     }
 
+    public AtlyssTextField AddTextField(ConfigEntry<string> config, string placeholder = "Placeholder") => AddTextField(config.Definition.Key, config, placeholder);
+
+    public AtlyssTextField AddTextField(string label, ConfigEntry<string> config, string placeholder = "Placeholder")
+    {
+        AtlyssTextField element = AddTextField(label, config.Value, placeholder);
+        element.OnValueChanged.AddListener(newValue => { config.Value = newValue; });
+        return element;
+    }
+
+    public AtlyssTextField AddTextField(string label, string value = "", string placeholder = "Placeholder")
+    {
+        AtlyssTextField element = TemplateManager.CreateTextField(Content);
+        Settings.OnCloseSettings.AddListener(() => element.Revert());
+        Settings.OnApplySettings.AddListener(() => element.Apply());
+
+        element.Label.text = label;
```

Comment:
`LabelText` shorthand property like prior
```suggestion
        element.LabelText = label;
```